src/main/java/org/gradle/profiler/mutations/ApplyChangeToAndroidLayoutFileMutator.java

```diff
@@ -0,0 +1,52 @@
+package org.gradle.profiler.mutations;
+
+import java.io.File;
+
+public class ApplyChangeToAndroidLayoutFileMutator extends AbstractFileChangeMutator {
+    public ApplyChangeToAndroidLayoutFileMutator(File sourceFile) {
+        super(sourceFile);
+    }
+
+    @Override
+    protected void applyChangeTo(StringBuilder text) {
+        // We need to handle both databinding layouts, which end with </layout>, and non, which can end with any layout item closing tag
```

Comment:
Can you replace all these inline comments with a class-level comment describing what this does instead?

Reply:
I meant to also remove the inline comments, as they go stale and are redundant with the method names :)

Reply:
Ah sorry I misunderstood, let me update